### ISSUE TITLE
[16.0] Reconcile OCA, unable to add Tax with Manual Operation

### ISSUE THREAD

**Issue by yung-wang** [OCA/account-reconcile#612]
## Module
account_reconcile_oca

## Describe the bug
In the note [Manual operation] there is no option to select a Tax Code. Even if you create a reconciliation model with taxes, after reconciliation the tax lines are not created. So it is not an option to use reconciliation models. Also using these models, if you have a lot of tax codes, you have to create a lot of models. This it not the right way. Please add the option to add taxes. It is in Odoo standard.
 

## To Reproduce
16.0

1. See screenshots from CE and ENT
CE
![image](https://github.com/OCA/account-reconcile/assets/30628967/9475dd3d-fa97-409d-a8fc-77ee610d7de4)

ENT
![image](https://github.com/OCA/account-reconcile/assets/30628967/ea2a6c82-7b46-4045-a95b-68bcfdba19cf)

**Expected behavior**
Have an option with tax codes.

**Comment by ThijsvOers** [comment]
As a sidenote I want to add that on the tax line that is created after reconciliation the tax grid shuld also be added in column Tax grid (in order to add the line to the tax report).

**Comment by lk-eska** [comment]
amount_currency and currency_id is neeeded too. it is possible to set the amount & currency when creating but there is no way to change it.

**Comment by lk-eska** [comment]
@etobella have you seen this issue?

**Comment by etobella** [comment]
No, I didn't see it, sorry. IMO, it is not an issue, it is a feature.

Obviously, it is interesting and so on, can you propose a PR for it?

**Comment by lk-eska** [comment]
it was about to close and i think these are needed. i cannot send a pr very soon but surely i can.
